<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>egwindow</class>
 <widget class="QDialog" name="egwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Sans</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButtonLightGang">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Gang</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightEss">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Ess</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightWohn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Wohn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightKueche">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Küche</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightKuecheWand">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Küche Wand</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightSpeis">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Speis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightArbeit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Arbeit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Zurück</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightVorraum">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Vorraum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightWC">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>WC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightTerrasse">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Terrasse</string>
   </property>
  </widget>
  <zorder>pushButtonLightGang</zorder>
  <zorder>pushButtonLightEss</zorder>
  <zorder>pushButtonLightWohn</zorder>
  <zorder>pushButtonLightKueche</zorder>
  <zorder>pushButtonLightSpeis</zorder>
  <zorder>pushButtonLightArbeit</zorder>
  <zorder>pushButtonBack</zorder>
  <zorder>pushButtonLightVorraum</zorder>
  <zorder>pushButtonLightWC</zorder>
  <zorder>pushButtonLightTerrasse</zorder>
  <zorder>pushButtonLightKuecheWand</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonBack</sender>
   <signal>clicked()</signal>
   <receiver>egwindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ioState(ioState*)</slot>
 </slots>
</ui>
